source/adapters/level_zero/v2/command_list_manager.cpp

```diff
@@ -13,11 +13,12 @@
 #include "../ur_interface_loader.hpp"
 #include "context.hpp"
 #include "kernel.hpp"
+#include "queue_handle.hpp"
```

Comment:
Why do we need to include this header if we're not using the `queue_handle_t_` type anymore?

Reply:
Good spot, removed it.